analysis/Scripts/FitScript.py

```diff
@@ -4,8 +4,10 @@
 from ROOT import TChain, TFile
 from Universal_Mass_Fit_function import Shape_fit
 
-user = "Simon"
-shape = "Bukin" # select the shape that you want to fit your data with
+#user = "Simon"
+user = input("please indicate the user (Simon, Chris or Nikhef): ")
+#shape = "Bukin" # select the shape that you want to fit your data with
+shape = input("please indicate the PDF (GaussCB or Bukin): ")
```

Comment:
Is this really beneficial? If you want to run it 10 times to fix some bugs, you have to manually enter your name 10 times...